README.md

```diff
@@ -78,6 +78,9 @@ within our community. ❤️
 - [Vulnerability disclosure](#vulnerability-disclosure)
 - [License](#license)
 
+## Community
+For a place to have open discussions on features, voice your ideas, or get help with general questions please visit our community at [Forem.dev](https://forem.dev/).
```

Comment:
**non-blocking question/suggestion:** Since the Community Name used in forem.dev is lowercased it maybe worth listing it that same way here? Or even consider update the Community name ([forem.dev](https://forem.dev) vs [Forem.dev](https://forem.dev))

![Screen Shot 2021-01-11 at 12 39 19](https://user-images.githubusercontent.com/6045239/104224231-50fb8880-540a-11eb-958c-05a575f89b07.png)

Reply:
Good point! Going to bring this up internally in forem.team since I am likely to change this in a few places over the next week. I think capitalizing would be a better option (purely based on how it looks 😅) but I don't have a strong preference for either.

Reply:
This is the kind of thing that we should clarify in our internal branding. I'd go with all lowercase for now (I personally think URLs should be described this way 😄)